game/world/objects/npc.cpp

```diff
@@ -2975,7 +2975,7 @@ bool Npc::rotateTo(float dx, float dz, float step, bool noAnim, uint64_t dt) {
 
   const auto sgn = std::sin(double(da)*M_PI/180.0);
   if(sgn<0) {
-    setAnimRotate( 1);
+    setAnimRotate(1);
```

Comment:
nitpick: space

Reply:
I removed the space. You want to keep it?